app/src/main/java/com/example/firstapplication/Activity/CryptoHomepageActivity.java

```diff
@@ -0,0 +1,126 @@
+package com.example.firstapplication.Activity;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.content.ContextCompat;
+import androidx.viewpager.widget.ViewPager;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+
+import com.example.firstapplication.Adapter.MpagerAdapter;
+import com.example.firstapplication.R;
+
+public class CryptoHomepageActivity extends AppCompatActivity implements View.OnClickListener {
+    private ViewPager mPager;
+    private int layouts[] = {
+            R.layout.crpyto_firstslide,R.layout.crypto_secondslide,
+            R.layout.crypto_thirdslide};
+    private MpagerAdapter mpagerAdapter;
+
+    //sliding dots
+    private LinearLayout dots_layout;
+    private ImageView dots[];
+
+    //buttons
+    private Button buttonNext, buttonSkip;
```

Comment:
nit: rename buttonNext and buttonSkip to btnNext and btnSkip

Reply:
okay